src/applications/facility-locator/components/SearchResultMessage.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+function SearchResultMessage({
```

Comment:
I know we are not consistent, but let's use es6 syntax for new code:

```suggestion
const SearchResultMessage = ({
 message,
  error,
  resultsFound,
  resultRef,
  facilityType
  }) => {
```